logchange-cli/pom.xml

```diff
@@ -0,0 +1,309 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <!-- MY I HAVE YOUR ATTENTION PLEASE -->
+    <!-- https://repo1.maven.org/maven2/io/micronaut/micronaut-parent/3.6.1/micronaut-parent-3.6.1.pom -->
```

Comment:
@witx98  ? XD

Reply:
I'm still cleaning up this PR, but actually, this is your comment 😄 @marwin1991